src/components/Button.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+
+function Button({ name }) {
```

Comment:
- [ ] [OPTIONAL] It will be nice if you use the arrow function as you are using ES6 syntax